proxy/src/proxy.rs

```diff
@@ -126,6 +132,13 @@ impl LatencyTimer {
         }
     }
 
+    pub fn pause(&mut self) -> LatencyTimerPause<'_> {
+        let now = Instant::now();
+        self.extra += now - self.start;
+        self.start = now;
```

Comment:
This line looks a bit unclear for me. Ideally, when the timer is paused, we shouldn't look at `start` field.
I would suggest one of the following options:
1. Do nothing.
2. Change start to `Option<Instant>`
3. Assign to start some clearly invalid value (I thought about UNIX_EPOCH, but in Instant there is no easy way to do that).

Reply:
2 is probably best. I updated start such that if all goes wrong, it at least doesn't over-count the recorded duration